packages/internet-header/src/components/post-internet-header/post-internet-header.tsx

```diff
@@ -182,6 +187,7 @@ export class PostInternetHeader {
     try {
       state.projectId = this.project;
       state.stickyness = this.stickyness;
+      state.selfAdminOrigin = this.selfAdminOrigin;
```

Comment:
Is it necessary/smart to handle the `selfAdminOrigin` different than the `logoutUrl` here?